superset-frontend/src/views/CRUD/welcome/ChartTable.tsx

```diff
@@ -59,6 +61,9 @@ function ChartTable({
   showThumbnails,
 }: ChartTableProps) {
   const history = useHistory();
+  const filterStore = getFromLocalStorage(HOMEPAGE_CHART_FILTER, null);
+  const defaultFilter = filterStore?.tab || 'Mine';
```

Comment:
```suggestion
  const defaultFilter = filterStore?.tab || TableTabTypes.MINE;
```

Can you replace other strings with constants/enums as well? There are a few occurences of hardcoded strings like 'Favourite' or 'Mine'